mycroft/configuration/config.py

```diff
@@ -216,6 +225,17 @@ def load_config_stack(configs=None, cache=False):
         # Merge all configs into one
         base = {}
         for c in configs:
+
+            # handle system constraints
+            if isinstance(c, RemoteConf):
+                if skip_remote:
+                    continue
+                c = {k: v for k, v in c.items() if k not in protected_remote}
```

Comment:
I may be reading this wrong, but this will only handle top level keys. For things like the `listener` we likely want to allow `listener.wake_word` but protect others like `listener.sample_rate`